HashMapSort/src/main/java/com/AndrieievYurii/UniqueCharCounter.java

```diff
@@ -0,0 +1,43 @@
+package com.AndrieievYurii;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class UniqueCharCounter {
+
+    static Map<String, HashMap<Character,Integer>> cachedWords = new HashMap<>();
+
+    public static HashMap<String, HashMap<Character, Integer>> getCachedWords() {
+        return (HashMap<String, HashMap<Character, Integer>>) cachedWords;
+    }
+
+    @Override
```

Comment:
you do not need toString method in utility class. It`s mostly for Entity classes where you have only state but not behaviour